requirements.dev.txt

```diff
@@ -6,3 +6,9 @@ sphinx
 sphinx_rtd_theme
 sphinxcontrib-plantuml
 autoapi
+ipdb
+-e git+https://github.com/HPENetworking/topology.git@master#egg=topology
+-e git+https://github.com/ocelotl/topology_docker_openswitch.git@master#egg=topology_docker_openswitch
+-e git+https://github.com/ocelotl/topology_docker_host.git@master#egg=topology_docker_host
```

Comment:
It could be, but it kinda defeats the purpose of removing all the node code out of this repo...